build/org.eclipse.pde.build/src/org/eclipse/pde/internal/build/AbstractScriptGenerator.java

```diff
@@ -255,17 +255,7 @@ protected static AntScript newAntScript(String scriptLocation, String scriptName
 		try {
 			@SuppressWarnings("resource") // intentional return AntScript without closed stream
 			OutputStream scriptStream = new BufferedOutputStream(new FileOutputStream(scriptLocation + '/' + scriptName));
-			try {
-				return new AntScript(scriptStream);
-			} catch (IOException e) {
-				try {
-					scriptStream.close();
-				} catch (IOException e1) {
-					e.addSuppressed(e1);
-				}
-				String message = NLS.bind(Messages.exception_writingFile, scriptLocation + '/' + scriptName);
-				throw new CoreException(new Status(IStatus.ERROR, PI_PDEBUILD, EXCEPTION_WRITING_FILE, message, e));
-			}
+			return new AntScript(scriptStream);
```

Comment:
This does not look equivalent.
In case the constructor of `AntScript` throws the `scriptStream` used to be closed. This isn't the case anymore and a potential resource leak.

Reply:
OK, `new AntScript()` never throws an `IOException`. So that's fine.
Sorry for the noise and thanks for this clean-up.